storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -2176,10 +2176,17 @@ decrypt_decompress_file(const char *filepath, uint thread_n)
 {
 	std::stringstream cmd, message,input_file,output_file;
 	bool needs_action = false;
+	int env_error;
 
         char *dest_filepath = strdup(filepath);
 	input_file << "xbinputfile" << thread_n;
-	setenv(input_file.str().c_str(), filepath, 1);
+	env_error = setenv(input_file.str().c_str(), filepath, 1);
+	if (env_error != 0) {
+		msg("Can not set env %s: got error %d\n",
```

Comment:
Minor fixes here:
 - Can we use msg_ts
 - Can we follow the standard of other errors that print the thread nr issuing the msg
 - Can we also set which file we were processing (`dest_filepath`)
 - Variable `env_error` will always be -1 in case of error.  `setenv` also sets [errno](https://man7.org/linux/man-pages/man3/errno.3.html) . A better user experience in case of error will be to return the actual error, rather than always -1.
 - With above, we don't need to store the return of `setenv`, just test it within the if clause.